### PR TITLE
DOCSP-41130: bulk write operations

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin-sync/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-41130>
Staging - https://preview-mongodbrustagir.gatsbyjs.io/kotlin-sync/DOCSP-41130-bulk/write/bulk-write/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/write/bulk-write.txt [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+- `bulkWrite() <{+api+}/mongodb-driver-kotlin-sync/com.mongodb.kotlin.client/-mongo-collection/bulk-write.html>`__
+- `DeleteManyModel <{+core-api+}/com/mongodb/client/model/DeleteManyModel.html>`__
+- `DeleteOneModel <{+core-api+}/com/mongodb/client/model/DeleteOneModel.html>`__
+- `InsertOneModel <{+core-api+}/com/mongodb/client/model/InsertOneModel.html>`__
+- `ReplaceOneModel <{+core-api+}/com/mongodb/client/model/ReplaceOneModel.html>`__
+- `UpdateManyModel <{+core-api+}/com/mongodb/client/model/UpdateManyModel.html>`__
+- `UpdateOneModel <{+core-api+}/com/mongodb/client/model/UpdateOneModel.html>`__
+- `BulkWriteOptions <{+core-api+}/com/mongodb/client/model/BulkWriteOptions.html>`__
+- `BulkWriteResult <{+core-api+}/com/mongodb/bulk/BulkWriteResult.html>`__
```
Comment (mcmorisi) [inline source/write/bulk-write.txt:344]: Complete nitpick here, but I think we should list the singular operations before the many operations.
```suggestion
- `bulkWrite() <{+api+}/mongodb-driver-kotlin-sync/com.mongodb.kotlin.client/-mongo-collection/bulk-write.html>`__
- `DeleteOneModel <{+core-api+}/com/mongodb/client/model/DeleteOneModel.html>`__
- `DeleteManyModel <{+core-api+}/com/mongodb/client/model/DeleteManyModel.html>`__
- `InsertOneModel <{+core-api+}/com/mongodb/client/model/InsertOneModel.html>`__
- `ReplaceOneModel <{+core-api+}/com/mongodb/client/model/ReplaceOneModel.html>`__
- `UpdateOneModel <{+core-api+}/com/mongodb/client/model/UpdateOneModel.html>`__
- `UpdateManyModel <{+core-api+}/com/mongodb/client/model/UpdateManyModel.html>`__
- `BulkWriteOptions <{+core-api+}/com/mongodb/client/model/BulkWriteOptions.html>`__
- `BulkWriteResult <{+core-api+}/com/mongodb/bulk/BulkWriteResult.html>`__
```

**Review by mcmorisi — APPROVED** [review]
Really great work, just a few small things. LGTM otherwise!

File: source/write/bulk-write.txt (hunk 1 above)
Comment (rustagir) [reply source/write/bulk-write.txt:344]: SUre, this was alphabetical but I will change

**Review by rozza — APPROVED** [review]
LGTM - one side comment about the wierd API url.

File: snooty.toml [hunk 2]
```diff
@@ -27,7 +27,7 @@ full-version = "{+version-number+}.2"
 version = "v{+version-number+}"
 mdb-server = "MongoDB Server"
 stable-api = "Stable API"
-api = "https://mongodb.github.io/mongo-java-driver/{+version-number+}/apidocs/mongodb-driver-kotlin-sync"
+api = "https://mongodb.github.io/mongo-java-driver/{+version-number+}/apidocs/mongodb-driver-kotlin-sync/mongodb-driver-kotlin-sync"
```
[the remark this answers is not in the document]
Comment (rustagir) [reply snooty.toml:30]: I'll look into this and re-deploy the Kotlin API docs!

[the remark this answers is not in the document]
Comment (rustagir) [reply snooty.toml:30]: https://jira.mongodb.org/browse/DOCSP-42238

**Pull request merged**